delivery_carrier_label_postlogistics/models/company.py

```diff
@@ -11,6 +17,11 @@ class ResCompany(models.Model):
     postlogistics_wsdl_url = fields.Char(compute='_get_wsdl_url',
                                          string='WSDL URL',
                                          store=True)
+    postlogistics_version = fields.Selection(
+        selection=POSTLOGISTICS_VERSIONS,
+        string="Version",
+        default=POSTLOGISTICS_VERSIONS[0][0],
```

Comment:
seems this should be a required field, maybe migration with initializing would be appropriate here :thinking:

Reply:
@i-vyshnevska the default value will be set when the field is created, and the field is not available on the company form. Maybe the required field could be added on the configuration panel to not allow the user to set the False value.
I say that because adding required fields on such common data model causes some trouble in unit tests which create companies, and force to add this module as dependency of the others modules creating company records.
What do you think?